ephios/core/templates/core/fragments/shift_box_big.html

```diff
@@ -19,7 +19,7 @@
         </div>
     </div>
 
-    <div class="card-footer">
+    <div class="card-footer no-print">
```

Comment:
add for disposition button as well

Reply:
Thought about moving that to the card-footer.